docker-compose.dev.yml

```diff
@@ -0,0 +1,10 @@
+# use as `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up`
+# this causes the the local folder to be loop-mounted into the container
+# so that you can edit code with your normal editors and have the changes
+# immediately be reflected in the running container, without a `docker-compose build`.
+
+version: '3'
+services:
+  web:
+    volumes:
+      - .:/code
```

Comment:
We don't need this file and we should keep using docker-compose.override.yml because for developers is easier to do `docker-compose up -d` than `docker-compose -f docker-compose.yml -f docker-compose.dev.yml`